openapi/paths/actors/acts.yaml

```diff
@@ -87,9 +87,9 @@ get:
   deprecated: false
 post:
   tags:
-    - Actors
-  summary: Create Actor
-  description: >-
```

Comment:
The new version actually works as proper Markdown. We just need to clean up the description and format them properly.

Reply:
Let's make it a separate issue.

It doesn't appear to be an issue for every `operation`. Based on that, I think it's caused by the manual edits after the programmatic migration. There's nothing I can do about it apart from editing them all manually. I think it's connected to #72  @Jkuzz tried to address. I'd keep that issue/PR as tracking for future reference and as a checklist item for the final rollout. 

I'll bring @TC-MO  up to speed since he might also be interested in reviewing the text and the formatting before we pull the plug on the original version.

Reply:
Agreed. From my PoV, it's actually good that we fixed the Markdown rendering, even if it technically "breaks" the visuals.

Reply:
Also it might be worth considering to introduce some automation to remove the double `\n`.